.github/workflows/tests.yml

```
@@ -1,6 +1,8 @@
 name: Tests
 
-on: [push]
+on:
+  push:
+  pull_request:
```

see: https://docs.github.com/en/github-ae@latest/actions/using-workflows/triggering-a-workflow#using-multiple-events
```suggestion
on: [push, pull_request]
```